EarTrumpet.Appx/layout/AppxManifest.xml

```diff
@@ -42,10 +42,10 @@
         DisplayName="EarTrumpet"
         Square150x150Logo="Assets\Square150x150Logo.png"
         Square44x44Logo="Assets\Square44x44Logo.png"
-        Description="EarTrumpet.UWP"
+        Description="EarTrumpet"
```

Comment:
@bebora: you might want to rebase your fork onto the latest `File-New-Project:master`. This commit should already be on the main trunk and shouldn't need to appear in this diff; not sure why it _does_ exist here to be honest unless Git history was rewritten on the main master to remove this commit after you forked the repository... Any idea how this ended-up on this PR, @GoldenTao?

Reply:
My best guess is... git happens?  No we had a few weird issues with code in Master that shouldn't have been and did a bit of work to split it back out, so there is a good chance depending on where you forks happened, they would have hit our rewriting of history.

Reply:
Oh my god, I need to get _Git happens..._ on a t-shirt 😆 Fair enough, in which case, yeah, you'll need to rebase your fork onto the latest head of master on the main repository otherwise this will reinstate something that was intentionally reverted by the repo maintainers.